kt/godot-library/src/main/kotlin/godot/util/Nullable.kt

```diff
@@ -0,0 +1,3 @@
+package godot.util
+
+inline fun <reified T> isNullable(): Boolean = null is T
```

Comment:
Not sure if we should expose this utility method. I'd either make it internal or if it is used in another module, I'd move that into the tools-common module